src-java/kilda-pce/src/main/java/org/openkilda/pce/finder/PathFinder.java

```diff
@@ -34,19 +33,19 @@
      *
      * @return a pair of ordered lists that represents the path from start to end, or an empty list if no path found.
      */
-    Pair<List<Edge>, List<Edge>> findPathInNetwork(AvailableNetwork network,
-                                                   SwitchId startSwitchId, SwitchId endSwitchId,
-                                                   WeightFunction weightFunction)
+    FindPathResult findPathInNetwork(AvailableNetwork network,
+                                     SwitchId startSwitchId, SwitchId endSwitchId,
+                                     WeightFunction weightFunction)
             throws UnroutableFlowException;
 
     /**
      * Finds a path whose weight is less than maxWeight and as close to maxWeight as possible.
      *
```

Comment:
Add info about `backUpMaxWeight` to javadoc please.

Reply:
Added.